pkg/apiutil/transport.go

```diff
@@ -51,6 +51,15 @@ func LoggingErrorEncoder(logger logger.Logger, enc kithttp.ErrorEncoder) kithttp
 		switch {
 		case errors.Contains(err, ErrBearerToken),
 			errors.Contains(err, ErrMissingID),
+			errors.Contains(err, ErrMissingThingID),
+			errors.Contains(err, ErrMissingProfileID),
+			errors.Contains(err, ErrMissingGroupID),
+			errors.Contains(err, ErrMissingGroupMemberID),
+			errors.Contains(err, ErrMissingOrgID),
+			errors.Contains(err, ErrMissingOrgMemberID),
```

Comment:
Use `ErrMissingMemberID` instead of `ErrMissingOrgMemberID`and `ErrMissingGroupMemberID`